clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesApiClientConfig.groovy

```diff
@@ -0,0 +1,112 @@
+package com.netflix.spinnaker.clouddriver.kubernetes.security
+
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.util.Config;
+import io.kubernetes.client.util.KubeConfig;
+import io.kubernetes.client.util.SSLUtils;
+
+import groovy.util.logging.Slf4j
+
+import javax.net.ssl.KeyManager;
+
+@Slf4j
+public class KubernetesApiClientConfig extends Config {
+  String kubeconfigFile
+  Boolean serviceAccount
+  String context
+  String cluster
+  String user
+  List<String> namespaces
+
+
+  public KubernetesApiClientConfig(String kubeconfigFile, String context, String cluster, String user, List<String> namespaces, Boolean serviceAccount) {
+    this.kubeconfigFile = kubeconfigFile
+    this.serviceAccount = serviceAccount
+    this.context = context
+    this.cluster = cluster
+    this.user = user
+    this.namespaces = namespaces
+  }
+
+  public ApiClient getApiCient() throws Exception {
+    if (kubeconfigFile == null) {
+      return Config.defaultClient()
+    } else {
+      return parseConfig(kubeconfigFile)
+    }
+  }
+
+  public ApiClient parseConfig(String fileName) throws IOException {
+    return this.parseConfig((Reader)(new FileReader(fileName)));
+  }
+
+  public ApiClient parseConfig(InputStream stream) {
+    return parseConfig((Reader)(new InputStreamReader(stream)));
+  }
+
+  public ApiClient parseConfig(Reader input) {
```

Comment:
I try to us the util.Config, but I don't know if we are going to handle any additional fields or not.  Also, I am not sure if we need apiVersion, namespace, ... which we discuss it before.  I can change just use util.Config.